torchgeo/datamodules/levircd.py

```diff
@@ -35,50 +44,90 @@ def __init__(
             patch_size: Size of each patch, either ``size`` or ``(height, width)``.
                 Should be a multiple of 32 for most segmentation architectures.
             num_workers: Number of workers for parallel data loading.
+            val_patch_sampling: Strategy for validation patch sampling:
+                'random' (default): Use RandomCrop, matches training distribution.
+                'grid': Use deterministic grid patches via ExtractPatches.
             **kwargs: Additional keyword arguments passed to
                 :class:`~torchgeo.datasets.LEVIRCD`.
         """
+        stride = kwargs.pop('stride', None)
```

Comment:
I don't think `stride` is documented? Do we need it? Shouldn't we always set stride == patch size to avoid double counting pixels?